cmd/ping/udp.go

```diff
@@ -137,17 +141,17 @@ func (u *udpServer) rateLimitedEchoResponse(b []byte, sender net.Addr) {
 	if vis.limit.Allow() {
 		b = bytes.TrimRight(b, "\x00")
 		if _, err := u.conn.WriteTo(b, sender); err != nil {
-			u.logger.WithError(err).Error("error sending returning udp packet")
+			u.logger.WithError(err).Error("Error sending returning udp packet")
 		}
 	} else {
-		logger.WithField("addr", sender.String()).Warn("rate limited. No response sent.")
+		logger.WithField("addr", sender.String()).Warn("Rate limited. No response sent")
```

Comment:
Should the last period be ommited, right?

Reply:
That certainly matches the style in this file (although no where else do I see multiple sentences, so maybe "Rate limited, no response set" would be more consistent).

Reply:
I found two examples in the current codebase.

* https://github.com/googleforgames/agones/blob/v1.15.0/cmd/allocator/main.go#L168
    * Uses `;` (the file itself looks almost old style though.)
* https://github.com/googleforgames/agones/blob/v1.15.0/cmd/controller/main.go#L118
    * Uses `.`.

I think the whole consistency would be better to be fixed in another PR, how about you?

Reply:
Yeah, this format doesn't bother me much. I'm more concerned with initial capitalisation. 

@roberthbailey any major objections to approving and merging? Otherwise I'm happy to merge on my end.

Reply:
This is an improvement over what we had before, so I'm happy with merging it and working more towards consistency in a follow up PR.